safe-global/safe-wallet-web
# Analytics: track number of owned safes and watchlist safes from the login page

## What it solves
The `Total added Safes' event was being tracked too much. Replace it with two events to track the number of safes in the watchlist and the number of owned safes. 

## How this PR fixes it
- From the login page (/welcome/accounts) emit two events to track the number of safes:
  - `Total Safes owned`
  - `Total Safes watchlist`
- Emit these events only once when logging in.
- Remove the `Total Added Safe` event

## How to test it
- visit `/welcome/accounts` 
- Check that the two events are emitted in the console.
- Check that the number of safes specified is correct in each case. The count is tracked in the `eventLabel` field
- Check that these events are not emitted elsewhere (such as when opening the sidebar from the console.)
- Check that `Total Added Safe` is no longer emitted anywhere.

## Screenshots
<img width="1512" alt="image" src="https://github.com/safe-global/safe-wallet-web/assets/17801424/360dc5af-c58c-417c-9aea-ae07e1c7df55">
